angrmanagement/__main__.py

```diff
@@ -9,7 +9,11 @@
 import platform
 import signal
 
-from . import __version__
+if sys.platform.startswith("darwin"):
+    from Foundation import NSBundle  # pylint: disable=import-error
+
+from . import __version__  # pylint: disable=wrong-import-position
```

Comment:
Let's put this above the conditional import so the linter is happy

Reply:
I don't mind, I ignored the linter since the convention was 'this package' import last, but I can rearrange them.

Reply:
I always put conditional imports after other imports, they're outside of the standard rules since they have logic associated with them.